SplitFiles/customer.c

```diff
@@ -159,7 +159,7 @@ void customer_func(GameComponent* customers, GameParameters* params) {
     }
 
     if (params->reputation != initialRep || 1 == g_mustRedraw) {
-        screenFace(18, 1, params->reputation, g_mustRedraw);
+        screenFace(18, 1, params->reputation - params->minReputation, g_mustRedraw);
```

Comment:
This is the main change - won't get sacked from neutral face any more, but likely to go from neutral to sad between end of Day X and start of Day X+1